docs/src/browsers.md

```diff
@@ -82,9 +82,27 @@ class Program
 }
 ```
 
-:::note
-Playwright bundles a recent Chromium build, but not Google Chrome or Microsoft Edge browsers - these should be installed manually before use.
-:::
+Note that Playwright bundles a recent Chromium build, but not Google Chrome or Microsoft Edge browsers - these should be installed manually before use. You can use Playwright command line interface to install the browsers:
```

Comment:
Would be good to reference there to the dedicated CLI page like we do in the other places. (Search for command line)

Reply:
done